README.md

```diff
@@ -4,6 +4,8 @@
 
 > This is a website that allows users to like and post reviews of movies they have previously watched.
 
+[Application screenshot](src/asset/app_screenshot.png)
+
```

Comment:
- [ ]  [OPTIONAL ] I believe putting the screenshot itself would be a great description for people visiting for the first time instead of putting a link. Kingly consider this suggestion.